src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlConnectionString.cs

```diff
@@ -556,12 +599,19 @@ internal SqlConnectionString(string connectionString) : base(connectionString, G
                 throw SQL.AuthenticationAndIntegratedSecurity();
             }
 
+#if NETFRAMEWORK
             if (Authentication == SqlAuthenticationMethod.ActiveDirectoryIntegrated && (_hasUserIdKeyword || _hasPasswordKeyword))
             {
                 throw SQL.IntegratedWithUserIDAndPassword();
             }
+#else
+            if (Authentication == SqlClient.SqlAuthenticationMethod.ActiveDirectoryIntegrated && _hasPasswordKeyword)
+            {
+                throw SQL.IntegratedWithPassword();
+            }
+#endif
```

Comment:
I think currently it's like that, we can double check with @David-Engel if we should align them in both frameworks.

Reply:
@cheenamalhotra @DavoudEshtehari 
In general, we are trying to avoid behavior changes. However, we also want to unify the code base, so we will have to evaluate each case. In this case, I assume user ID is simply ignored in the netcore code and I think we should consolidate on that less restrictive behavior since that is less likely to break any existing netcore connection strings.